resources/user-agents/browsers/boxee/boxee.json

```diff
@@ -7,6 +7,7 @@
     {
       "userAgent": "Boxee",
       "device": "Boxee",
+      "platform": "Linux",
```

Comment:
All the patterns here were defined as "Linux" platform, so I just moved it up here.  I had to leave it for the one that has `#PLATFORM#` in the match though.